data/ohn.yaml

```diff
@@ -0,0 +1,18 @@
+description: This definition extracts the O/H/N submechanism from GRI-Mech 3.0
```

Comment:
This is hand-copied from `nasa.cti` in 89157a4 (same for many other instances) ... would it make sense to add parsing of the header block to `ctml2yaml`? (this would also benefit users converting their own files)

Reply:
Yes, I think it would be good to add this functionality... I think it should be possible in `ctml2yaml`, I believe comments are accessible in the tree. I'm not sure about `cti2yaml`, because that actually `exec`s the input file, so I'm not sure if comments are available. I believe a docstring would be available and could be added though...

Reply:
From what I recall, CTI files typically have a commented `# ` header. The simplest way would be just to parse this block using `with open(...) as cti_file:` etc.?

Reply:
`ctml2yaml` doesn't really stand a chance here of having a relevant description to copy, since the comment header in the CTI file would have been lost when running `ctml_writer`, and there are very few XML files with any header comment blocks, at least in the `cantera` repository.

I spent a little time trying to get access to the comments in a CTI file when writing `cti2yaml`, but couldn't find a way to get at them in a way that provided correspondence to the code itself -- they don't seem to be available even through the `ast` module. If we're only after the header comment to use for this field, then I think @ischoegl's suggestion is probably the best we can do.

Reply:
@speth I think it's probably worth seeing how hard this would be to add to `ctml2yaml`, in case anyone has hand-edited XML files that include comments, at least for a top-level comment. Not a high priority, I'll add an issue at Cantera/enhancements eventually.